pkg/verifier/aws/entry_point.go

```diff
@@ -207,6 +208,45 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 
 	// Terminate the EC2 instance (unless user requests otherwise)
 	if !vei.SkipInstanceTermination {
+		//Replaced the SGs attached to the network-verifier-instance by the default SG in order to allow
+		//deletion of temporary SGs created
+
+		//Getting a list of the SGs for the current VPC of our instance
+		var defaultSecurityGroupID = ""
+
+		describeSGOutput, err := a.AwsClient.DescribeSecurityGroups(vei.Ctx, &ec2.DescribeSecurityGroupsInput{
+			Filters: []ec2Types.Filter{
+				{
+					Name:   awsTools.String("vpc-id"),
+					Values: []string{vpcId},
+				},
+				{
+					Name:   awsTools.String("group-name"),
+					Values: []string{"default"},
+				},
+			},
+		})
+		if err != nil {
+			a.Output.AddError(err)
+		}
+
+		//Filtering SGs to get only the default SG ID finishing the iteration as soon as we find the default SG.
+		for _, SG := range describeSGOutput.SecurityGroups {
+			if *SG.GroupName == "default" {
+				defaultSecurityGroupID = *SG.GroupId
+			}
+		}
```

Comment:
Did you mean to put a `break` here to finish the iteration early, as you state in the comment?

Reply:
I did but not needed anymore. it was a comment from my first try. The way is now the filter should return a unique SG anyways. Updating the comment